app-admin/etcd-wrapper/etcd-wrapper-0.0.3.ebuild

```diff
@@ -22,6 +22,8 @@ RDEPEND=">=app-emulation/rkt-1.9.1[rkt_stage1_fly]"
 S=${WORKDIR}
 
 src_install() {
-	dobin "${FILESDIR}"/etcd-wrapper
-  	systemd_dounit "${FILESDIR}"/etcd-wrapper.service
+	exeinto /usr/lib/coreos
+	doexe "${FILESDIR}"/etcd-wrapper
+
+  	systemd_dounit "${FILESDIR}"/etcd-member.service
```

Comment:
We can't call it `etcd.service` because that already exists (as much as I would have loved to replace it). What would you prefer we call it?